### ISSUE TITLE
Bug: $createThread

### ISSUE THREAD

**Issue by sr-star-trek** [aoijs/aoi.js#563]
### Bug Type

Function

### Bugged Function

```js
$createThread[1219764403898159114;Example!;60;public;$messageID;false]

1219764403898159114 - СhannelID - forum
```

### Version of aoi.js

v6.6.1

### Version of Node.js

v16.11.0 (Lowest)

### Relevant log output

```shell
AoiError: Invalid Failed To Create Thread With Reason: TypeError [GuildForumMessageRequired]: You must provide a message to create a guild forum thread  
 { 
   Function : `$createThread`,
   Command : `eval`,
   Version : 6.7.1 
 }
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Faf4a** [comment]
Thank you for reporting this! I can indeed confirm that this is a bug.

**Comment by Faf4a** [comment]
This was fixed in [a998471](https://github.com/AkaruiDevelopment/aoi.js/commit/a9984717dcdec5913e9ba2700596d67b415244a2)!
